.heroku/vendor/lib/libyajl.so.2

```diff
@@ -0,0 +1 @@
+libyajl.so.2.1.0
```

Comment:
To use the yajl backend I had to add the libyajl.so binary to our repo and instructions to use it.